flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -486,4 +630,14 @@ enum FlxAxes
 {
 	X;
 	Y;
+}
+
+enum GamepadModel
+{
+	Logitech;
+	OUYA;
+	PS3;
+	PS4;
+	Xbox;
```

Comment:
I'm just going strictly off of what I'm getting from the device.name from the Flash API. 

On flash target, I get nice useful strings that tell me what it is, "Microsoft X-Box 360 pad"
On legacy target, I get nothing, and just default to 360 b/c I have no choice.
On next target, I get "XInput device" and the numerical mappings are different than what worked for the XBox 360 in windows on legacy.

So maybe this is a conversation to have with @singmajesty about what the openfl-next gamepad API is returning for the device string names.